src/Form/CaseFilterType.php

```diff
@@ -41,9 +46,10 @@ class CaseFilterType extends AbstractType
      */
     private $userRepository;
 
-    public function __construct(BoardRepository $boardRepository, FilterHelper $filterHelper, TranslatorInterface $translator, UserRepository $userRepository)
+    public function __construct(BoardRepository $boardRepository, CaseEntityRepository $caseEntityRepository, FilterHelper $filterHelper, TranslatorInterface $translator, UserRepository $userRepository)
```

Comment:
Class constructor property promotion is your friend (cf. https://php.watch/versions/8.0/constructor-property-promotion).

Reply:
Fixed in 332d773517f7a733edd56475bf67262300969905.